app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,30 @@
+class RestaurantsController < ApplicationController
+
+  include Response
+  include ActiveModel
+  def index
+    @restaurants = Restaurant.all
+    # Cuisine.find(@restaurants.cuisine_id)
+    # render :json => @restaurants.to_json(:except => [:created_at, :updated_at], include: 'cuisine' )
+    render :json => @restaurants.to_json(:except => [:created_at, :updated_at, :cuisine_id], :include => {:reviews => {:except => [:created_at, :updated_at, :restaurant_id]}, :cuisine => {:except => [:created_at, :updated_at]}})
```

Comment:
I saw you've added the active_model_serializers gem but you're not using it, do you plan to later refactor this to use serializers? as what you did here is very similar to the purpose of serializers.

Reply:
tried first with a serializer then tried doing it inline ... don't know if in the next parts i'll use serialization functionalities though